facturacion_mexico/facturacion_fiscal/tests/test_ffm_summary_folio.py

```
@@ -0,0 +1,38 @@
+- como último recurso, muestra lo suelto.
+
+Seeding directo por DB (sin red).
+"""
+
+import frappe
+from frappe.tests import IntegrationTestCase
+
+from facturacion_mexico.api.ffm_summary import get_ffm_summary
+
+
+def _seed_ffm(**fields):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_seed_ffm`

(ANN202)